### PR TITLE
feat: Add Favourite event option in Event Details Fragment action bar

### LINKED ISSUE #166: Add favourite option in action bar in event details
Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #166 
screenshot_20180625-145201

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 1]
```diff
@@ -219,7 +234,6 @@ class EventDetailsFragment : Fragment() {
         bundle.putLong("EVENT_ID", eventId)
         ticketFragment.arguments = bundle
         activity?.supportFragmentManager?.beginTransaction()?.replace(R.id.frameContainer, ticketFragment)?.addToBackStack(null)?.commit()
-
```
[the remark this answers is not in the document]
Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: @simarsingh24 Removed this line feed as all other functions don't have a line feed between the last line of code and bracket

[the remark this answers is not in the document]
Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: cool

File: app/src/main/res/menu/event_details.xml [hunk 2]
```diff
@@ -14,4 +14,10 @@
         android:icon="@drawable/ic_share_white_24dp"
         android:title="@string/add_to_calendar"
         app:showAsAction="always"/>
+
+    <item
+        android:id="@+id/favoriteEvent"
+        android:icon="@drawable/ic_baseline_favorite_border_white_24px"
+        android:title="@string/add_to_calendar"
```
Comment (iamareebjamal) [inline app/src/main/res/menu/event_details.xml]: Change title

Comment (dreadpool2) [reply app/src/main/res/menu/event_details.xml]: Changed it :+1:

**Pull request merged**